packages/shared/src/components/modals/NewSourceModal.tsx

```diff
@@ -242,23 +248,35 @@ export default function NewSourceModal(props: ModalProps): ReactElement {
           data-testid={`login state: ${loginState?.trigger}`}
           id="submit-source"
         >
-          <SearchField
-            disabled={!!feeds?.length}
-            className="my-4"
+          <TextField
+            disabled={!isEnabled}
+            className={{
+              container: 'mt-4',
+            }}
             inputId="new-source-field"
             name="url"
+            label="Paste blog / RSS URL"
             placeholder="Paste blog / RSS URL"
-            showIcon={false}
             autoComplete="off"
             type="url"
             autoFocus
             aria-describedby={scrapeError && 'new-source-field-desc'}
             valueChanged={onUrlChanged}
             fieldType="primary"
-            rightButtonProps={false}
+            leftIcon={<LinkIcon />}
+            rightIcon={
+              !isEnabled ? (
+                <LockIcon className="text-theme-label-disabled" />
+              ) : undefined
+            }
+            style={{
+              // needed to overwrite fill from base.css input style
+              // to make the placeholder color work
+              WebkitTextFillColor: 'unset',
```

Comment:
Had to do it few times already, basically it is due to this, not sure why it was added https://github.com/dailydotdev/apps/pull/2533/files but it affects all inputs.

Reply:
@rebelchris I am still not sure what did it fix? Any idea how to solve on top level?

Reply:
We can maybe add this to `base.css`

```css
input::placeholder {
  -webkit-text-fill-color: var(--theme-label-quaternary);
}
```

To make it fixed for our placeholder color.

Reply:
Then I would not need `style` overrides.

Reply:
Yeah that's what I mean with top level fix :) 
For all I care we can ignore the other PR I literally can't recall why it was done or what it solved precisely.

Reply:
Haha, I was moving in that direction too, I'll remove it. From what I can see it fixes placeholder colors in bunch of different places as well.